### DISCUSSION TITLE
Tool to see what HW acceleration is available?

### DISCUSSION THREAD

**Discussion by jglazer63** [RandomEngy/VidCoder#951] [General]
Subject sorta says it all.  I have a Kaby Lake processor (Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz   3.70 GHz), albeit about 4 years old and an NVidea GeForce GTX 1050.  What hardware acceleration can I use?  Is there a tool out there that can tell me what my options are?

Thanks!

**Comment by RandomEngy** [comment]
If the hardware encoder shows up in the Video Codec list under the Video Encoding tab, then it's supported. For example my system only supports NVEnc: 
![image](https://user-images.githubusercontent.com/9450861/147395388-c7102f59-c4a4-463c-9cf8-6da81c8a5d02.png)

**Reply by jglazer63** [reply]
> If the hardware encoder shows up in the Video Codec list under the Video Encoding tab, then it's supported. For example my system only supports NVEnc: ![image](https://user-images.githubusercontent.com/9450861/147395388-c7102f59-c4a4-463c-9cf8-6da81c8a5d02.png)

Oh cool.   That helps a lot.  I wasn't sure which I could choose.  Thank you.
